### DISCUSSION TITLE
Linxu Kernel bug reports

### DISCUSSION THREAD

**Discussion by ZerBea** [ZerBea/hcxdumptool#469] [General]
As of today I have decided not to make any more bug reports on [bugzilla](https://bugzilla.kernel.org/)
Also I have closed all my open bug reports.

I thought [bugzilla](https://bugzilla.kernel.org/) was the central point of contact for bug reports.
Looks like this is not the case:
https://bugzilla.kernel.org/show_bug.cgi?id=219040#c11

**Comment by strasharo** [comment]
I would suggest everyone here to openly express their opinion regarding the behavior of such triggered individuals as that guy. I'll do it right now.

**Comment by ZerBea** [comment]
As an alternative, wireless bugs can be reported directly to the wireless mailing list. The entire procedure is explained here:
https://www.kernel.org/doc/html/v4.19/admin-guide/reporting-bugs.html

e.g. for mt76:
From inside the Linux Kernel tree run this pearl script to identify the maintainer:
`$ perl scripts/get_maintainer.pl -f <filename>`
```
$ perl scripts/get_maintainer.pl -f drivers/net/wireless/mediatek/mt76/Makefile
Felix Fietkau <nbd@nbd.name> (maintainer:MEDIATEK MT76 WIRELESS LAN DRIVER)
Lorenzo Bianconi <lorenzo@kernel.org> (maintainer:MEDIATEK MT76 WIRELESS LAN DRIVER)
Ryder Lee <ryder.lee@mediatek.com> (maintainer:MEDIATEK MT76 WIRELESS LAN DRIVER)
Shayne Chen <shayne.chen@mediatek.com> (reviewer:MEDIATEK MT76 WIRELESS LAN DRIVER)
Sean Wang <sean.wang@mediatek.com> (reviewer:MEDIATEK MT76 WIRELESS LAN DRIVER)
Kalle Valo <kvalo@kernel.org> (maintainer:NETWORKING DRIVERS (WIRELESS))
Matthias Brugger <matthias.bgg@gmail.com> (maintainer:ARM/Mediatek SoC support)
AngeloGioacchino Del Regno <angelogioacchino.delregno@collabora.com> (maintainer:ARM/Mediatek SoC support)
linux-wireless@vger.kernel.org (open list:MEDIATEK MT76 WIRELESS LAN DRIVER)
linux-kernel@vger.kernel.org (open list:ARM/Mediatek SoC support)
linux-arm-kernel@lists.infradead.org (moderated list:ARM/Mediatek SoC support)
linux-mediatek@lists.infradead.org (moderated list:ARM/Mediatek SoC support)
```
Than send your bug report.

**Comment by ZerBea** [comment]
I'm compiling kernel 6.11 to figure out if things are going better now.
I don't expect that everything is working flawless out of the box (especially in combination with BT and USB3), because the changes on mt76 and mac80211 are huge and the driver developers are very busy.
